interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1975,7 +1955,7 @@ private void checkCensus(MetricsRecord record, boolean server,
     for (MessageLite response : responses) {
       uncompressedResponsesSize += response.getSerializedSize();
     }
-    if (server && serverInProcess()) {
+    if (isServer && server != null) {
```

Comment:
can server be null is isServer is true?

Reply:
Yes. I did a refactor to check `server != null` before calling `checkCensus()`. This should be better.